desktop-sync/latest/using/select-to-sync.md

```diff
@@ -10,10 +10,19 @@ Once you've set up Desktop Sync, all your Alfresco Content Services folders are
 
 Use the *Choose files and folders to sync* screen to select the content to sync between Alfresco Content Services and your desktop. The content is synced at `C:\Users\<username>\Alfresco`.
 
+If you work with content that has deep folder structures, you may wish to enable the **Include all files and sub-folders** checkbox. This allows you to only sync selected folders deep in your hierarchy, without syncing the intermediate folders, and prevent lengthy folder paths being created in your local sync paths.
```

Comment:
Thanks for the feedback & follow-up discussion. Since the feature is **enabled** by default, I'll attempt to revise these paragraphs so that the description matches the use-case.